spiffymanagement.py

```diff
@@ -272,3 +272,48 @@ def questbackup(cfg, server):
         if isUp(server):
             log.info(f'{server} is running, STILL DON\'T CARE!')
         log.info('DONE!')
+
+        def ftbquestbackup(cfg, server):
+    """Aren't those quest mods great?"""
+
+    bpath = cfg['backupspath']
+    if server not in cfg['servers']:
+        log.warning(f'{server} has been misspelled or not configured!')
+    elif 'worldname' not in cfg['servers'][server]:
+        log.warning(f'{server} has no world directory specified!')
+    else:
+        world = cfg['servers'][server]['worldname']
+        log.info(f'Starting backup for {server}\'s ftbquests...')
+        if isUp(server):
+            log.info(f'{server} is running, don\'t care, just want FTBQUESTS!')
+        sbpath = f'{bpath}/{server}/ftbquests'
+        try:
+            os.makedirs(sbpath, exist_ok=True)
+        except Exception as e:
+            log.error(e + '\nBackup aborted, DANGER! might loose quests!')
+            return False
+        else:
+            log.info('Created missing directories! (if they were missing)')
+        log.info('Deleting old quest backups...')
+        now = time()
+        with os.scandir(sbpath) as d:
+            for entry in d:
+                if not entry.name.startswith('.') and entry.is_file():
+                    stats = entry.stat()
+                    if stats.st_mtime < now - (10080 * 60):
+                        try:
+                            os.remove(entry.path)
+                        except OSError as e:
+                            log.error(e)
+                        else:
+                            log.info(f'Deleted {entry.path} for being too old!')
+        log.info('Creating ftbquest backup...')
+        bname = datetime.now().strftime('%Y.%m.%d-%H-%M-%S') + f'-{server}-{world}-FTBQUESTS.tar.gz'
+        os.chdir(sbpath)
+        serverpath = cfg['serverspath']
+        with tarfile.open(bname, 'w:gz') as tf:
+            tf.add(f'{serverpath}/{server}/{world}/data/ftb_lib', 'ftb_lib')
```

Comment:
This whole function is basically a duplicate of 'questbackup' with a different backup target; Why not just modify the questbackup function to accept an additional argument or two to achieve the same thing?